bundles/org.openhab.binding.iammeter/src/main/java/org/openhab/binding/iammeter/internal/IammeterConfiguration.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.iammeter.internal;
+
+/**
+ * The {@link IammeterConfiguration} class contains fields mapping thing configuration parameters.
+ *
+ * @author yang bo - Initial contribution
+ */
+
+public class IammeterConfiguration {
```

Comment:
Can you add the `@NonNullByDefault` annotation?

Reply:
added.